server/recipeAPI/IRecipeAPI.ts

```diff
@@ -0,0 +1,30 @@
+import IRecipe from "../serverAPI/model/recipe/IRecipe";
+
+export default interface IRecipeAPI {
+     /**
+     * Retrieves collection of partially filled Recipe items that satisfy searching parameters.
+     * 
+     * @param parameters parameters used for searching.
+     * - query - required parameter that defines the name of the Recipe item (partial names are accepted).
+     * - number - optional parameter that defines max number of the results to be returned.
+     * - page - optional parameter that defines page number for pagination.
```

Comment:
For all optional parameters, please specify what the default value is

Reply:
well, default is non... I just don't use them if they are not specified